app/app_lcl.h

```diff
@@ -22,7 +22,7 @@ extern "C"
  */
 #define DEFAULT_SERVER "127.0.0.1"
 #define DEFAULT_PORT 443
-#define DEFAULT_URI_PREFIX "acvp/v1/"
+#define DEFAULT_URI_PREFIX "/acvp/v1/"
```

Comment:
Wasn't this changed because of the double slashes ?

Reply:
Hey I changed this because every URL the serve hands us starts with /acvp/v1/ and the new acvp_transport_get function I made to handle constructed strings along with the ones the server hands us. It was easier to "fix" this default prefix to have / leading